src/hardware/input/mouse_config.cpp

```diff
@@ -459,34 +487,51 @@ static void config_init(Section_prop& secprop)
 
 	// Built-in DOS driver configuration
 
-	prop_bool = secprop.Add_bool("builtin_dos_mouse_driver", only_at_start, true);
-	assert(prop_bool);
-	prop_bool->Set_help(
-	        "Enable the built-in DOS mouse driver ('on' by default). This results in the\n"
-	        "lowest possible latency and the smoothest mouse movement, so only disable it\n"
-	        "and load a real DOS mouse driver if it's really necessary (e.g., if a game is\n"
-	        "not compatible with the built-in driver).\n"
-	        "  on:   Enable the built-in mouse driver. `ps2_mouse_model` and\n"
-	        "        `com_mouse_model` have no effect on the built-in driver.\n"
-	        "  off:  Disable the built-in mouse driver (if you don't want mouse support or\n"
-	        "        you want to load a real DOS mouse driver). To use a real DOS driver\n"
-	        "        (e.g., MOUSE.COM or CTMOUSE.EXE), configure the mouse type with\n"
-	        "        `ps2_mouse_model` or `com_mouse_model`, then load the driver.\n"
-	        "        A real DOS driver might increase compatibility with some programs,\n"
-	        "        but will introduce more input latency.\n"
-	        "Note: The built-in driver is auto-disabled if you boot into real MS-DOS or\n"
-	        "      Windows 9x under DOSBox. Under Windows 3.x, the driver is not disabled,\n"
-	        "      but the Windows 3.x mouse driver takes over.");
+	prop_str = secprop.Add_string("builtin_dos_mouse_driver",
+	                              only_at_start,
+	                              OptionBuiltInDosDriver::On);
+	assert(prop_str);
+	prop_str->Set_values({
+		OptionBuiltInDosDriver::Off,
+		OptionBuiltInDosDriver::On,
+		OptionBuiltInDosDriver::NoTsr
+	});
+	prop_str->Set_help(
+	        "Built-in DOS mouse driver mode ('on' by default). It bypasses the PS/2 and\n"
+	        "serial (COM) ports and communicates with the mouse directly. This results in\n"
+	        "lower input lag, smoother movement, and increased mouse responsiveness, so only\n"
+	        "disable it and load a real DOS mouse driver if it's really necessary (e.g. if a,\n"
+	        "game is not compatible with the built-in driver).\n"
+	        "  on:      Simulate a mouse driver TSR program loaded from AUTOEXEC.BAT\n"
+	        "           (default). This is the most compatible way to emulate the DOS mouse\n"
+	        "           driver, but if it  doesn't work with your game, try the 'no-tsr'\n"
+	        "           setting.\n"
+	        "  no-tsr:  Enable the mouse driver without simulating the TSR program. Let us\n"
+	        "           know if it fixes any software not working with the 'on' setting.\n"
+	        "  off:     Disable the built-in mouse driver. It can still be started in runtime\n"
+	        "           by manually running the MOUSE.COM utility available on drive Z.\n"
+	        "Notes:\n"
```

Comment:
"You can still start it at runtime by running the bundled MOUSE.COM from drive Z."

Reply:
Changed - but I've written "by executing", to avoid "run" appearing twice in close proximity.